product_listings_manager/__init__.py

```diff
@@ -1,35 +1,36 @@
-import traceback
-from flask import Flask
-from flaskext.xmlrpc import XMLRPCHandler, Fault
+__version__ = '0.0.1'
```

Comment:
I've tagged v0.1.0, so let's make `__version__` align with that: `'0.1.0'`

Reply:
Fixed.